README.md

```diff
@@ -142,7 +139,7 @@ Below is an example `lmeconfig.json`:
 }
 ```
 This configuration will overwrite the default configurations.
-So, 
+So,
```

Comment:
Looks like we've already a more recent version in master branch. We've to modify on the top of that. We'll handle this as a new issue.